.github/CODEOWNERS

```
@@ -0,0 +1,3 @@
+# Assigning ownership of all files to the ENSNode team
+
+* @namehash/ensawards
```

```suggestion
* @namehash/ensnode
```

There's no team on GitHub for "ensawards". But we have a team for "ensnode". For simplicity let's have the ENSNode team manage ENSAwards too.